packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/vsphere/esxiSecretFieldValidator.ts

```diff
@@ -43,7 +43,7 @@ const validateUser = (value: string): ValidationMsg => {
 
   if (value === '') {
     return {
-      type: 'error',
```

Comment:
lets add a new `defautl` state when `value === undefined` , that way we keep the `error` if the mandatory value is missing while adding a new nutral `defautl` state at form initiation, where the value is `undefined` and the validation will be `defatul` , WDYT?

Reply:
Sure, a good idea.